rfcs/0136-stabilize-incrementally.md

```diff
@@ -0,0 +1,386 @@
+
+## Difficulties in the roll-out
+
+Process concerns are addressed by having a clear process, with clear outcomes, before any stabilization is begun.
+That is the purpose of this RFC!
+
+## Flakes are criticized for eating other Nix features
+
+We do *not* propose adding delay to once-again separate those features from Flakes, as that would introduce more feature work which would delay stabilization and be unacceptable to the pro-Flakes faction.
+
+That said, an ancillary incremental beneift of incremental stabilization is to bolster a *sense* of layering in Nix's user interface that has been, according to some people, lost.
+Ensuring that these two subsets of the new CLI --- without Flakes, and without Nix language support --- do in fact make sense in isolation will provide a "scaffolding" upon which interested parties can later introduce generalized features like search and pure eval without Flakes.
+
+The hope is that such scaffolding will assuage this faction their concerns are heard without holding things up.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+The main downside is a delay from the process of splitting Nix up, and then a delay between the stabilization steps.
+
+The fact that [RFC 134] is already mostly implemented, and in code review, is hopefully reason enough to believe it shouldn't take much longer.
+That maximum delay waiting for that to complete should be dwarfed by duration of time we've spent "in limbo" without a clear plan to move forward.
+We therefore think that is a small and reasonable price to pay for the benefit of community harmony.
+
+An addition, in the detailed design there is an escape hatch saying that blocking on the implementation of [RFC 134] can be reconsidered if things do indeed take longer than foreseen.
+
+The delay of "auditing and refining" shouldn't represent time "idle" from a stabilization perspective.
+As long as we are making progress stabilizing features and having healthy discussions, we don't see any problem.
+
+**Step 0** is also designed to take the pressure off these possible sources of delay, giving us *something* to work on that is not blocked on [RFC 134] or anything else.
+
+# Alternatives
+[alternatives]: #alternatives
+
+We could, of course, just "rip off the band-aid" and stabilize everything at once.
+The argument for that would be that enough time has passed and the concerns of (less numerous) long-time users are not important.
+But we think the plan here has little downsides; we can instead make everyone happy with only some delay.
+If that is true, why not do that instead!
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+None at this time.
+
+# Future work
+[future]: #future-work
+
+Generalization features to work without Flakes, like pure eval and search, might be desired by the Flake-skeptic faction, but is purposely left as future work in order to not delay stabilization.
```

Comment:
I think that you meant to write “Generalizing” instead of “Generalization”. Also, I would move the parenthetical and replace its commas with parentheses to make the sentence easier to understand. In other words, I would change it to “Generalizing features (like pure eval and search) to work without Flakes might be desired by the Flake-skeptic faction[…]”